doc/PACKAGERS.md

```diff
@@ -0,0 +1,83 @@
+Packaging
+=========
+
+Thank you for taking the time to package Rizin!
+
+There are some things to consider when packaging Rizin for your distribution.
+We will look at each step below.
+
+Cloning Source
+--------------
+
+Rizin uses git submodules to track versions of dependencies. You can check out
+the Rizin source and all dependencies using
+`git clone --recursive https://github.com/rizinorg/rizin`.
+
+If you change branches run `git submodule update --init --recursive` to ensure
+you are tracking the correct versions of the submodules.
+
+Building
+--------
+
+Rizin uses the `meson` build system. Command line flags passed to meson can
+change how Rizin is built.
+
+To define the base install location for Rizin use the `--prefix` flag when
+invoking `meson`. For system installs it is common to use `/usr`. If in doubt
+check your distributions packaging guidelines.
+
+Rizin uses the Capstone disassembly engine and supports versions 3, 4, and 5.
+By default we pull version 4 and statically link it into the Rizin executables.
+Some distributions might prefer that a system version of Capstone be
+dynamically linked at runtime. To do this, use the `-Duse_sys_capstone=true`
+command line option when running `meson`. See `capstone.md` for more details.
+
+Once you are happy with the flags you have passed to `meson` to configure your
+build you need to actually compile Rizin using `ninja`. You can do this with
+`ninja -C build`.  The `-C` flag tells `ninja` to change into the directory
+`build` before building. If you do not specify a `target` the default target of
+`all` will be used.
+
+Packaging
+---------
+
+The final step is creating a package from the build outputs. To do this we will
```

Comment:
"To do this it may be required to invoke ... , based on the distribution you are targeting."

Not all distros actually require to set DESTDIR manually (e.g. for Fedora I don't need to do that).